trace-reader/src/test/java/org/hypertrace/trace/reader/AvroUtil.java

```diff
@@ -0,0 +1,51 @@
+package org.hypertrace.trace.reader;
+
+import java.nio.ByteBuffer;
+import java.util.Collections;
+import java.util.Map;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.Attributes;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.MetricValue;
+import org.hypertrace.core.datamodel.Metrics;
+import org.hypertrace.core.datamodel.StructuredTrace;
+
+public class AvroUtil {
+
+  public static StructuredTrace.Builder defaultedStructuredTraceBuilder() {
```

Comment:
Mmh how is this used?

Reply:
This is a test util, used to build structured traces to pass into the reader in various tests.

Reply:
ok